practicesharp/Equation_Validator.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace basic_calculator
+{
+    class Equation_Validator
+    {
+        public string equation_s { get; private set; }
+
+        public Equation_Validator(string userinput)
+        {
+            equation_s = "";
+            if (Validate(userinput)) equation_s= userinput;
+        }
+        public bool Validate(string word)
+        {
+            try
+            {
+                const string allowed_letters = "1234567890+-*/";
```

Comment:
Should be passed via the constructor.
This will allow you to change the chars set later without touching this class.
Also, this class is working now, but it will not work for << operator which has 2 chars and for floating numbers (1.5, 2.3 etc) because "2. + 3" is not valid, it must be "2.0 + 3"